Public/Src/Sandbox/MacOs/Interop/Posix/memory.c

```diff
@@ -42,15 +42,58 @@ int GetRamUsageInfo(RamUsageInfo *buffer, long bufferSize)
     return KERN_SUCCESS;
 }
 
+static uint64_t ProcessTreeWorkingSetSize(pid_t pid, const rlim_t max_proc_count, bool *success)
+{
+    uint64_t children_usage = 0;
+
+    pid_t pids[max_proc_count];
+    int child_count = proc_listchildpids(pid, pids, (int) max_proc_count);
+    *success &= child_count >= 0;
+
+    for (int i = 0; (i < child_count) && *success; i++)
+    {
+        int pid = pids[i];
```

Comment:
Symbol `pid` is already defined as one of the function arguments (`pid_t pid`). How does this even compile?  Please rename

Reply:
Yeah, thats a C quirk that allows to overwrite definitions in local scopes. I renamed for clarity.